template.rb

```diff
@@ -92,23 +92,33 @@ def copy_solidus_starter_frontend_files
   copy_file 'config/initializers/devise.rb', 'tmp/devise.rb'
   append_file 'config/initializers/devise.rb', File.read('tmp/devise.rb')
 
-  application <<~RUBY
-    if Rails.autoloaders.main
-      # Rails 7
+  if Rails.gem_version >= Gem::Version.new('7')
```

Comment:
Are we sure that will work for Rails 6 with Zeitwerk configured?

https://guides.rubyonrails.org/classic_to_zeitwerk_howto.html#why-switch-from-classic-to-zeitwerk-questionmark